src/vivarium/framework/components/manager.py

```diff
@@ -1,8 +1,11 @@
-"""The ``vivarium`` component management system.
-
-This module contains the interface and the default implementation of the ``vivarium`` component manager.
-The manager is responsible for tracking all components in the system and for initiating the ``setup``
-life-cycle stage of each component.
+"""This module contains the interface and the default implementation of the
+``vivarium`` component manager system.
+
+The component manager is a plugin that manages all of the components in a
+simulation and is responsible for the ``setup`` phase of their lifecycle.  It
+maintains a distinction between the top-level (or "manager") components that are
```

Comment:
We've been using "top-level" components to refer to those specified directly in the model specification.  We should be consistent.